server/src/main/java/com/linecorp/centraldogma/server/internal/storage/repository/MirrorConfig.java

```diff
@@ -115,6 +115,12 @@ public MirrorConfig(@JsonProperty("enabled") @Nullable Boolean enabled,
     @Nullable
     Mirror toMirror(Project parent, Iterable<MirrorCredential> credentials) {
         if (!enabled || localRepo == null || !parent.repos().exists(localRepo)) {
+            final String localRepoStatus = localRepo == null ? "not specified" : "does not exist";
+            logger.warn("Mirroring is not possible because one or more conditions are not met: " +
+                            "mirroring enabled={}, localRepo={}, localRepo status={}. " +
+                            "Please ensure mirroring is enabled, a local repository name is specified, " +
+                            "and the local repository exists.",
+                    enabled, localRepo == null ? "null" : localRepo, localRepoStatus);
```

Comment:
It should be noisy to warn for a disabled mirror. How about leaving the warning for the  `enabled` mirror configurations?

Reply:
Let's completely remove this warning. This log is only present in the server, so users wouldn't see this warning log anyway. I believe we need to address this in https://github.com/line/centraldogma/pull/838

Reply:
Understand, thanks for explain to me